Release 1.12.0 (#796)
astronomer/astronomer-providers: CHANGELOG.rst

Changelog
=========

1.12.0 (2022-12-02)
-------------------

Feature
"""""""

- Add custom XCom backend for GCS.
  (`#681 <https://github.com/astronomer/astronomer-providers/pull/681>`_)

Bug Fixes
"""""""""

> - Change return value of ``SnowflakeOperatorAsync`` to be same as the Sync operator.

Reviewer: ```suggestion
- Change return value of ``SnowflakeOperatorAsync`` to be same as that of ``SnowflakeOperator``
```

Author: Fixed

  (`#781 <https://github.com/astronomer/astronomer-providers/pull/781>`_)
- Add poke_interval to ``S3KeySensorAsync``.
  (`#782 <https://github.com/astronomer/astronomer-providers/pull/782>`_)

> - Fix ``AwsBaseHookAsync`` to get connection details using ``AwsConnectionWrapper``.

Reviewer: this sentence need to be from a user point of view

Author: Use ``AwsConnectionWrapper`` to get connection details to create AWS async client.

  (`#758 <https://github.com/astronomer/astronomer-providers/pull/758>`_)

> - Remove mypy ignore specification in ``WasbHookAsync``.

Reviewer: not needed as this is minor tech. change

Author: Ok, removing it
